### DISCUSSION TITLE
less don't support arbitrary value about shadow

### DISCUSSION THREAD

**Discussion by wqcstrong** [tailwindlabs/tailwindcss#11508] [Help] [answered]
**What version of Tailwind CSS are you using?**

v3.3.2

**What happened**

I want to use shadow arbitrary values in less like this:
```css
/* ./tailwind-init.less */

@layer components {
  .theme-shadow {
    @apply shadow-[0_0_14px_rgba(200,200,0,1)];
  }
}
```
 but the less seems not support the syntax, is there some ways to fix it?

**Reproducation link**

- ✅ CSS: https://play.tailwindcss.com/hUUu0nhMx7?file=css
- ❌ Less: https://lesscss.org/less-preview/#eyJjb2RlIjoiXG5AbGF5ZXIgY29tcG9uZW50cyB7XG4gIC50aGVtZS1zaGFkb3cge1xuICAgIEBhcHBseSBzaGFkb3ctWzBfMF8xNHB4X3JnYmEoMjAwLDIwMCwwLDEpXTtcbiAgfVxufSIsImFjdGl2ZVZlcnNpb24iOiI0LjEuMyJ9

**Comment by wongjn** [comment, marked as answer]
Consider escaping the `@apply` value: https://lesscss.org/less-preview/#eyJjb2RlIjoiXG5AbGF5ZXIgY29tcG9uZW50cyB7XG4gIC50aGVtZS1zaGFkb3cge1xuICAgIEBhcHBseSB+XCJzaGFkb3ctWzBfMF8xNHB4X3JnYmEoMjAwLDIwMCwwLDEpXVwiO1xuICB9XG59IiwiYWN0aXZlVmVyc2lvbiI6IjQuMS4zIn0=

**Reply by wqcstrong** [reply]
It's cool~ Thank you, bro!
